### DISCUSSION TITLE
Immich Android App - just for backup?

### DISCUSSION THREAD

**Discussion by GD-Dal** [immich-app/immich#12325] [Q&A]
I downloaded the Immich app for Android to look at pictures from my local server running a Immich docker instance.
But I wasn't able to do it.
It seems that the Immich app on Android is meant to create backups from the phone, and not to picture collections from a server?

Am I correct in this assumption?

Thanks

**Comment by bo0tzz** [comment]
No, you should be fine to use the app only for viewing as well. The first sync might take a bit of time though - look for the loading spinner in the bottom left.
